src/joystick/psp/SDL_sysjoystick.c

```diff
@@ -210,7 +210,7 @@ static void PSP_JoystickUpdate(SDL_Joystick *joystick)
     static enum PspCtrlButtons old_buttons = 0;
     static unsigned char old_x = 0, old_y = 0;
 
-    sceCtrlReadBufferPositive(&pad, 1);
+    if(sceCtrlPeekBufferPositive(&pad, 1) <= 0) return;
```

Comment:
Can you fix the formatting for SDL style? Otherwise this change looks good to me.

Reply:
just adding a space after the if ? (sorry, i cant run clang-format as i run a custom distro based on musl libc, and getting clang to compile requires 3 dozen patches)

Reply:
```c
    if (sceCtrlPeekBufferPositive(&pad, 1) <= 0) {
        return;
    }
```

Reply:
thanks, fixed